learn/getting_started/supported_os.md

```diff
@@ -0,0 +1,25 @@
+# Supported operating systems
+
+Please ensure your operating system is listed on this page before installing the Meilisearch binary. Refer to the [changelog](https://github.com/meilisearch/MeiliSearch/releases) for more information on operating system support changes.
+
+If the provided [binaries](/learn/getting_started/quick_start.md#local-installation) don't work for you, we recommend compiling Meilisearch from the source files. If the compilation fails, Meilisearch is not compatible with your machine.
+
+::: warning
+Any operating system or version not listed on this page does not mean Meilisearch won't work with it at all, we just don't guarantee it through our test suite.
+:::
+
```

Comment:
I tried to integrate the info here on the first paragraph. The tone here is too informal for technical documentation, IMHO.

```suggestion
```